crates/phactory/src/lib.rs

```diff
@@ -297,14 +297,6 @@ impl<Platform: pal::Platform> Phactory<Platform> {
     }
 
     pub fn init(&mut self, args: InitArgs) {
-        if args.git_revision != git_revision() {
```

Comment:
What's this feature for?

Reply:
It was from pruntime v0, to ensure the `app` and `enclave.so` are compiled from the same codebase.

Reply:
OK